corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -326,6 +326,6 @@ $.fn.reportUserConfigurableConfigEditor = function (options) {
 $.fn.reportConfigList = function (options) {
     this.each(function(i, v) {
         var viewModel = new ReportConfigsViewModel(options);
-        ko.applyBindings(viewModel, $(this).get(i));
+        $(this).koApplyBindings(viewModel);
```

Comment:
I'm inclined to update these functions to not loop at all, but only ever act on one object, as I think that's typical of our jQuery extensions. But I can see an argument for keeping the loop, since you could theoretically be dealing with multiple elements.